### DISCUSSION TITLE
No hint that there are assets for a release

### DISCUSSION THREAD

**Discussion by rbeeger** [community/community#6034] [Other Feature Feedback, Questions, & Ideas] [answered]
Previously the latest release in the list of releases had an assets section that was expanded. You could read about the release and instantly knew what assets were available. Now you don't know whether there are any assets for a release. You have to open the release itself to see the list of assets. Especially for open source apps that provide installers this is confusing. You might get the impression that the only way to get the app is to clone the repository and build it yourself.

**Comment by MylesBorins** [comment, marked as answer]
This came in https://github.com/github/feedback/discussions/5962#discussioncomment-1426476 and we already have a scoped work item to fix this.

Thank you for the feedback!

**Reply by MylesBorins** [reply]
We shipped a fix this week FYI
